### ISSUE TITLE
not working on opera next 23

### ISSUE THREAD

**Issue by cukabeka** [joelspadin/SitePreferences#4]
I just tried to set site specific preferences (= enable flash) here:
http://22tracks.com/ams/house/49173

on right-click, no menu entry for SitePreferences is shown. 

I'm using Opera Next 23 /Mac and have a bunch of extensions installed.

**Comment by Piter432** [comment]
This is known bug from Opera 22 as far I can remember. The developer of this extension is not sure, what cause it. Maybe it's Opera bug or maybe he need to investigate his extension deeper.

**Comment by joelspadin** [comment]
What version of the extension are you using? There is an Opera bug where the chrome.runtime.onInstalled event is not fired when the browser starts up (I have verified using Chrome that this event should be fired) and so the extension never gets a chance to set itself up. I added a bit of a hack to the latest version which works around this, so if you do have the latest version, there is another issue here.
